wolfcrypt/test/test.c

```diff
@@ -6259,6 +6259,7 @@ static int ecc_test_vector_item(const eccVector* vector, WC_RNG* rng)
     memset(sig, 0, sizeof(sig));
     x = sizeof(sig);
 
+    wc_ecc_free(&userA);  /* free key before importing a new one over the top */
```

Comment:
That's another way to fix it for sure!  I was thinking maybe the make key / check key pair was for testing different sizes.  Maybe not?